zbench.zig

```diff
@@ -219,13 +214,69 @@ pub const Benchmark = struct {
 /// It takes a pointer to a Benchmark object.
 pub const BenchFunc = fn (*Benchmark) void;
 
-/// BenchmarkResult stores the result of a single benchmark.
-/// It includes the name and the total duration of the benchmark.
+/// BenchmarkResult stores the resulting computed metrics/statistics from a benchmark
 pub const BenchmarkResult = struct {
-    /// Name of the benchmark.
     name: []const u8,
-    /// Total duration of the benchmark in nanoseconds.
-    duration: u64,
+    percs: Percentiles,
```

Comment:
Add brief comments for each field, especially for `percs`, `avg_duration`, `std_duration`, etc., to explain their purpose and how they are calculated or used. Thats needed for the generated documentation.